tests/lean/semicolonOrLinebreak.lean.expected.out

```diff
@@ -0,0 +1,5 @@
+building '/nix/store/kzswn86bwsdhxg5jb5d8vnfjkq36akl7-lean.drv'...
```

Comment:
This is probably a typo.

Reply:
Thanks, I finally made sure this doesn't happen to me all the time...